README.md

```diff
@@ -63,7 +63,28 @@ async function getActivity() {
             limit: 10,
             page: 1,
         })
-        console.log(data)
+
+        const activities = data.data
+
+        for (const activity of activities) {
+            console.log(activity?.id)
+
+            const activityMetadata = activity?.metadata
+
+            if (activityMetadata?.__typename === "NetflixActivityMetadata") {
+                console.log(activityMetadata.title)
+                console.log(activityMetadata.date)
+    
+                const activityMetadataSubject = activityMetadata.subject
+                if (activityMetadataSubject) {
```

Comment:
@TosinJs there's no way a subject won't exist. I don't think you need to include this whole section about the subject.

Reply:
@timigod From the graphql schema on staging, Subject is not a required field. But I have removed that section though.